src/index.js

```diff
@@ -1,7 +1,135 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Map from './Map';
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
+
+
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = { 
+      markerPosition: { lat: this.randomLat(), lng: this.randomLng() },
+      score: 100
+    };
+ }  
+
+
+
+  
+randomLat () {
+  let lat = Math.random() * (45.00541896831666 - 42.730315121762715) + 42.730315121762715;
+  return(lat);
+}
+randomLng (){
+  let lng = (Math.random() * (71.51022535353107 - 73.35218221090553) + 73.35218221090553) * -1;
+return (lng);
+}
+  
+  moveMarkerNorth = () => {
```

Comment:
This is a good first pass at the movement buttons. After you finish the rest of the stories in the backlog you might want to come back and refactor these functions into a single function since all of these methods are very similar.